pkg/chain/ethereum/ecdsa/gen/abi/WalletRegistry.go

```diff
@@ -28,6 +28,20 @@ var (
 	_ = event.NewSubscription
 )
 
+// EcdsaDkgData is an auto generated low-level Go binding around an user-defined struct.
```

Comment:
The regenerated contract bindings include dApp-only functions. The contracts should be generated based on the mainnet bindings so you need to run `make mainnet`. The default environment is `development`:

https://github.com/keep-network/keep-core/blob/86e7644fbe7de9fc801437f8a70844546b20c21a/Makefile#L1-L5

With the contracts regenerated based on the mainnet artifacts we should see no changes in other bindings that the ones for `MaintainerProxy` contract.

Reply:
Indeed, after running `make mainnet` there are only changes related to `MaintainerProxy`.
Done in 36c0c90b775a81a366f20c8e40a396ab4c10af92.